festim/temperature/temperature_solver.py

```diff
@@ -8,7 +8,7 @@ class HeatTransferProblem(festim.Temperature):
     Args:
         transient (bool, optional): If True, a transient simulation will
             be run. Defaults to True.
-        initial_condition (festim.InitialCondition, optional): The initial condition.
+        initial_condition (int, float, sp.Expr, festim.InitialCondition, optional): The initial condition.
```

Comment:
Won't it be useful to add a small test to check `create_functions` with `sp.Expr`?

Reply:
Can do

Reply:
Done! Let me know what you think

Reply:
Perfect